src/main/kotlin/com/terraformation/backend/accelerator/db/DefaultVoterStore.kt

```diff
@@ -0,0 +1,32 @@
+package com.terraformation.backend.accelerator.db
+
+import com.terraformation.backend.customer.model.requirePermissions
+import com.terraformation.backend.db.accelerator.tables.references.DEFAULT_VOTERS
+import com.terraformation.backend.db.default_schema.UserId
+import jakarta.inject.Named
+import org.jooq.DSLContext
+
+@Named
+class DefaultVoterStore(
+    private val dslContext: DSLContext,
+) {
+  fun fetch(userId: UserId? = null): List<UserId> {
```

Comment:
Splitting this into `fetch()` and `exists(userId)` (the latter just returning a boolean) would make the intent clearer at call sites. Maybe rename `fetch()` to `findAll()` to follow the jOOQ DAO naming convention which other store classes tend to do for fetch functions.